pkg/services/org/orgimpl/store_test.go

```diff
@@ -542,20 +570,21 @@ func TestIntegration_SQLStore_GetOrgUsers(t *testing.T) {
 		{
 			desc: "should return all users",
 			query: &org.SearchOrgUsersQuery{
-				OrgID: 1,
+				OrgID: o.ID,
 				User: &user.SignedInUser{
-					OrgID:       1,
+					OrgID:       o.ID,
 					Permissions: map[int64]map[string][]string{1: {accesscontrol.ActionOrgUsersRead: {accesscontrol.ScopeUsersAll}}},
 				},
 			},
-			expectedNumUsers: 10,
+			// out of the 10 created users all but the 1st are added to the organization
+			expectedNumUsers: 9,
```

Comment:
not sure what's exactly the purpose of this block but it was already there so I have kept it:
https://github.com/grafana/grafana/blob/323a4f9499ebfd65640933ca9a8d690081612400/pkg/services/org/orgimpl/store_test.go#L637-L644

Reply:
I _think_ the first user is added as an admin somewhere else maybe?

Reply:
hmm 🤔  the deleted `CreateUserForTests` used to do that:
https://github.com/grafana/grafana/blob/3468804ff5f3eb542df414680e1838c4183d05ea/pkg/services/user/userimpl/user.go#L426-L446
thank you, I will fix the test appropriately to restore the old behaviour